Added Pi Network (PI) token metadata

SharifovAkmal opened the pull request:
This PR adds metadata for the Pi Network (PI) token, including:
- Smart contract ID
- Token symbol, name, and description
- Image URL and admin address

Please review and approve. Thank you!

SharifovAkmal committed:
Added Pi Network (PI) token metadata

SharifovAkmal commented:
Hi, I received an email requesting a 10 TON payment for token verification.
Can you confirm if this is an official requirement from TON Keeper?

I want to make sure this is legitimate before proceeding.

Thank you!

Gene090 commented:
Hi. Be careful, reviewing your request is completely free and there is no need to pay any fee.

Gene090 commented:
Is there any information about the project? Is there a website? Social media?
Also, don't add your own points to the file, don't change the 'address' line name.

SharifovAkmal committed:
Updated Pi Network metadata with project details

SharifovAkmal commented:
I have updated the file based on your feedback:
- Changed `id:` to `address:`
- Added website and social media links
- Removed unnecessary fields

Please review and let me know if further changes are needed. Thanks!

Gene090 commented:
Hello. Could you please clarify if there is any mention of this token on the official website?

SharifovAkmal committed:
Update PI.yaml

SharifovAkmal commented:
Hi,

Upon reviewing the official Pi Network website, there is no mention of a Pi token on the TON blockchain. However, there is a community-driven project called PITON that integrates Pi concepts with the TON blockchain.

I've updated the `PI.yaml` file to reflect this community initiative. Please review the changes and let me know if further information is needed.

Thank you!

SharifovAkmal commented:
Hi, my PR has received an approval from `trng2002`, but I see that it still requires maintainer approval.
Could you please review and approve it when possible?
Thanks!

salateur commented:
Hello. trng2002 is not a part of our team, so we do not know w/e it is.
After reviewing your PR we've decided to decline verification:
- you do not have enough holders.
- no liquidity
- nothing in particular about the token at all: no roadmaps, nothing.

Verification is not required. Develop your token and gather strong community. Then return to us.

trngco reviewed (commented):
Wait for me, <3!,
Cảm ỏn vì đã giúp đõ!,

trngco commented:
Dear, your friend!,Waitting for me!, Thanks!, Sent from my iPhoneOn 15-03-2568 BE, at 05:16, salateur ***@***.***> wrote:﻿
Closed #2700.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because your review was requested.Message ID: ***@***.***>

trngco commented:
Dear, your friend!, 
Watting for me!,
Thanks!,

Sent from my iPhone

> On 08-03-2568 BE, at 04:33, Gene090 ***@***.***> wrote:
> 
> Hello. Could you please clarify if there is any mention of this token on the official website?

trngco commented on jettons/PI.yaml:
```diff
@@ -0,0 +1,10 @@
+address: "0:5b122c6491246663a148e4e6a7442f1b3856f4a5c5a25872439bfc1e3c84b726"
+name: "PITON"
+symbol: "PITON"
+decimals: 9
+description: "PITON is a community-driven token designed for enthusiasts of both Pi Network and the TON blockchain."
+image: "https://piton.network/logo.png"
+website: "https://piton.network/"
+social:
+  - "https://twitter.com/pitonnetwork"
+  - "https://t.me/pitoncommunity"
```
I agree!,

trngco reviewed (commented):
<3!,

trngco commented on jettons/PI.yaml:
```diff
@@ -0,0 +1,10 @@
+decimals: 9
+description: "PITON is a community-driven token designed for enthusiasts of both Pi Network and the TON blockchain."
+image: "https://piton.network/logo.png"
+website: "https://piton.network/"
```
I gree!,

trngco commented:
> Hi, I received an email requesting a 10 TON payment for token verification.
> Can you confirm if this is an official requirement from TON Keeper?
> 
> I want to make sure this is legitimate before proceeding.
> 
> Thank you!
Yes!,
